### PR TITLE
migrate additional election stats

### LINKED ISSUE #529: Validators - List of TODOS
- [x] validator version bar, and the count, needs to show staked only
- [x] add election time (in min) (1h), st. dev
- [x] add election time (24h), st. dev
- [x] add election times (7d), st. dev
- [x] add time since last election
image

### PR DESCRIPTION
**Pull request by allenan**
closes:
- https://github.com/helium/explorer/issues/529
- https://github.com/helium/explorer/issues/532

### CODE REVIEW

File: hooks/useApi.js [hunk 1]
```diff
@@ -2,6 +2,15 @@ import useSWR from 'swr'
 
 export const API_BASE = 'https://explorer-api.helium.com/api'
 
+const fetcher = async (url) => {
+  const response = await fetch(url, {
+    headers: {
+      'cache-control': 'max-age=60',
```
Comment (allenan) [inline hooks/useApi.js:8]: permits cloudflare to cache explorer-api responses up to 60 seconds

Comment (danielcolinjames) [reply hooks/useApi.js:8]: oh nice!

**Review by danielcolinjames — APPROVED** [review]
looks great! the /elections tab is a nice addition, and makes sense to have it on /blocks and /validators 👍

**Comment by allenan** [comment]
> looks great! the /elections tab is a nice addition, and makes sense to have it on /blocks and /validators 👍

yeah wasn't sure where best to put that, but figured it applied to both

**Pull request merged**